examples/kubernetes/groupsnapshot-v1beta1.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: groupsnapshot.storage.k8s.io/v1alpha1
+apiVersion: groupsnapshot.storage.k8s.io/v1beta1
```

Comment:
Can you fix the label?
`groupsnapshot.storage.k8s.io/volumeGroupSnapshotName: postgresql`

Reply:
Hi! If you are referring to `app.kubernetes.io/name: postgresql`, that's just the selector this example uses to get the PVC list. I don't think we need to change it 👍